capstone/src/components/User/Adventurer/Adventurer.jsx

```diff
@@ -0,0 +1,25 @@
+import './Adventurer.css';
+import { useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
+import FindAdventure from './FindAdventure/FindAdventure';
+import Header from '../Header/Header';
+
+export default function Adventurer({ setIsLoggedIn, isLoggedIn }) {
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (!isLoggedIn) {
+      navigate('/');
+    }
+  }, [isLoggedIn]);
```

Comment:
This is fine for now, but if you find yourself doing this a lot, you might want to create a **parent component**  that wraps around all logged in views.